js/add-books.js

```diff
@@ -0,0 +1,49 @@
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const add = document.querySelector('.submit');
+const allBooks = document.querySelector('#all-books');
+const error = document.querySelector('.error');
+
+const bookStore = JSON.parse(localStorage.getItem('books')) || [];
+
+function showBooks() {
+  for (let i = 0; i < bookStore.length; i += 1) {
+    allBooks.innerHTML += `
+    <article class="books">
+      <p>${bookStore[i].title}</p>
+      <p>${bookStore[i].author}</p>
+      <button class="remove" onclick="removeBook(${i})">Remove</button>
+      <hr>
+    </article>
+`;
+  }
+}
```

Comment:
Great job team🤝

- [ ] When adding the second book to the collection, an extra book is rendered to the UI. Kindly fix this to make your project conform to the project requirements.

Reply:
Hello @1GORDON 

Thanks a lot for your review..

So please, How do you think we can solve this problem cause we saw it but we try to fix it but it isn't work, hope you can help us with that, please?